utils/emmylua/wesnoth/units.lua

```diff
@@ -247,6 +247,16 @@ function animator:clear() end
 ---@param params unit_animator_params
 function animator:add(unit, flag, hits, params) end
 
+---Fakes the move of a given unit, can be queued.
+---Todo: move it to C++ side implementation, add continuous movement functionality.
+---@param u_a unit_animator
+---@param u unit
+---@param to_x integer
+---@param to_y integer
+---@param scroll boolean
+---@overload fun(u_a:unit_animator, u:unit, target:location, scroll:boolean)
+function animator:add_movement(u_a, u, to_x, to_y, scroll) end
```

Comment:
Almost there. Just remove the `u_a` parameter – that's implied by the use of a colon here.

Reply:
The documentation is fine now, yes.